protocol/daemons/pricefeed/client/client_integration_test.go

```diff
@@ -446,9 +453,10 @@ func (s *PriceDaemonIntegrationTestSuite) TestUpdateMarkets_AddMarketWithUSDTCon
 	)
 }
 
-// TestUpdateMarkets_InvalidUpdates tests that the pricefeed daemon ignores invalid updates.
-func (s *PriceDaemonIntegrationTestSuite) TestUpdateMarkets_InvalidUpdate() {
-	// Start the daemon with a single market. Then, update the endpoint to return invalid params.
+// TestUpdateMarkets_PartialUpdates tests that the pricefeed daemon applies valid market params and discards invalid
+// params whenever an update is partially valid.
+func (s *PriceDaemonIntegrationTestSuite) TestUpdateMarkets_PartialUpdate() {
```

Comment:
Since the entire update is no longer valid/invalid, I updated the test to check for the application of valid updates when some param updates are invalid.